modular_skyrat/modules/customization/datums/dna.dm

```diff
@@ -236,6 +236,9 @@ GLOBAL_LIST_EMPTY(total_uf_len_by_block)
 			language_holder = new species_holder(src)
 		update_atom_languages()
 
+		//Signal to handle situations when teshari rotates
+		if(mrace == /datum/species/teshari)
```

Comment:
```suggestion
		if(istype(mrace, /datum/species/teshari)
```

Reply:
I'm not sure about this... It's bloks registration of the signal.

Reply:
The definition of mrace is datum/species/mrace. 
So... This commit was an mistake.